pkg/core/tokens/validator.go

```diff
@@ -2,14 +2,19 @@ package tokens
 
 import (
 	"context"
+	errors2 "errors"
+	"fmt"
 	"strconv"
 
 	"github.com/golang-jwt/jwt/v4"
 	"github.com/pkg/errors"
 
 	store_config "github.com/kumahq/kuma/pkg/config/core/resources/store"
+	"github.com/kumahq/kuma/pkg/core"
 )
 
+var log = core.Log.WithName("tokens-validator")
```

Comment:
I'd pass it to `NewValidator` with indication what type of token it is. Something like
```
NewValidator(..., ..., core.Log.WithName("tokens-validator").WithName("zone-token"))
```